tests/unit/PolicyUtilsTest.ts

```diff
@@ -81,4 +91,60 @@ describe('PolicyUtils', () => {
             });
         });
     });
+    describe('getFilteredPolicies', () => {
+        beforeAll(() => {
+            Onyx.init({
+                keys: ONYXKEYS,
+                initialKeyStates: {
+                    // Given mock data for session
+                    [ONYXKEYS.SESSION]: {accountID: CARLOS_ACCOUNT_ID, email: CARLOS_EMAIL},
+                },
+            });
+        });
+
+        beforeEach(() => {
+            global.fetch = TestHelper.getGlobalFetchMock();
+            return Onyx.clear().then(waitForBatchedUpdates);
+        });
+        it('should return empty array', () => {
+            // Given mock data for policies
+            const policies = {
+                // eslint-disable-next-line @typescript-eslint/naming-convention
+                1: {
+                    ...createRandomPolicy(1, CONST.POLICY.TYPE.CORPORATE),
+                    role: '',
+                },
+            };
+            // When calling getFilteredPolicies with policies
+            const result = PolicyUtils.getFilteredPolicies(policies as OnyxCollection<Policy>);
+            // The result should be empty array since the policies contains only one policy which does not have role field.
+            expect(result.length).toBe(0);
+        });
+        it('should return array contains policy which has id = 1', () => {
+            // Given mock data for policies
+            const randomPolicy1 = createRandomPolicy(1, CONST.POLICY.TYPE.CORPORATE);
+            const policies = {
+                // eslint-disable-next-line @typescript-eslint/naming-convention
+                1: randomPolicy1,
+            };
+            // When calling getFilteredPolicies with policies
+            const result = PolicyUtils.getFilteredPolicies(policies as OnyxCollection<Policy>);
+            // The result should contains the policy which has id = 1 since it is a valid policy.
+            expect(result).toContainEqual(randomPolicy1);
+        });
+        it('should return empty array', () => {
+            // Given mock data for policies
```

Comment:
Please update like my other suggestions - the `Given` comment should explain the condition so it can't be so generic as "mock data".